draft-marx-qlog-h3-events.md

```diff
@@ -0,0 +1,931 @@
+area: Transport
+workgroup: QUIC
+keyword: Internet-Draft
+
+stand_alone: yes
+pi: [toc, docindent, sortrefs, symrefs, strict, compact, comments, inline]
+
+author:
+  -
+    ins: R. Marx
+    name: Robin Marx
+    org: KU Leuven
+    email: robin.marx@kuleuven.be
+  -
+    ins: L. Niccolini
+    name: Luca Niccolini
+    org: Facebook
+    email: lniccolini@fb.com
+    role: editor
+  -
+    ins: M. Seemann
+    name: Marten Seemann
+    org: Protocol Labs
+    email: martenseemann@gmail.com
+    role: editor
+
+normative:
+
+  QUIC-HTTP:
+    title: "Hypertext Transfer Protocol Version 3 (HTTP/3)"
+    date: {DATE}
+    seriesinfo:
+      Internet-Draft: draft-ietf-quic-http-latest
+    author:
+      -
+          ins: M. Bishop
+          name: Mike Bishop
+          org: Akamai Technologies
+          role: editor
+
+  QUIC-QPACK:
+    title: "QPACK: Header Compression for HTTP over QUIC"
+    date: {DATE}
+    seriesinfo:
+      Internet-Draft: draft-ietf-quic-qpack-latest
+    author:
+      -
+          ins: C. Krasic
+          name: Charles 'Buck' Krasic
+          org: Google, Inc
+      -
+          ins: M. Bishop
+          name: Mike Bishop
+          org: Akamai Technologies
+      -
+          ins: A. Frindell
+          name: Alan Frindell
+          org: Facebook
+          role: editor
+
+  QLOG-MAIN:
+    title: "Main logging schema for qlog"
+    date: {DATE}
+    seriesinfo:
+      Internet-Draft: draft-marx-qlog-main-schema-latest
+    author:
+      -
+        ins: R. Marx
+        name: Robin Marx
+        org: KU Leuven
+        role: editor
+      -
+        ins: L. Niccolini
+        name: Luca Niccolini
+        org: Facebook
+        role: editor
+      -
+        ins: M. Seemann
+        name: Marten Seemann
+        org: Protocol Labs
+        role: editor
+
+  QLOG-QUIC:
+    title: "QUIC event definitions for qlog"
+    date: {DATE}
+    seriesinfo:
+      Internet-Draft: draft-marx-qlog-quic-events-latest
+    author:
+      -
+        ins: R. Marx
+        name: Robin Marx
+        org: KU Leuven
+        role: editor
+      -
+        ins: L. Niccolini
+        name: Luca Niccolini
+        org: Facebook
+        role: editor
+      -
+        ins: M. Seemann
+        name: Marten Seemann
+        org: Protocol Labs
+        role: editor
+
+informative:
+
+--- abstract
+
+This document describes concrete qlog event definitions and their metadata for
+HTTP/3 and QPACK-related events. These events can then be embedded in the higher
+level schema defined in [QLOG-MAIN].
+
+--- middle
+
+# Introduction
+
+This document describes the values of the qlog name ("category" + "event") and
+"data" fields and their semantics for the HTTP/3 and QPACK protocols. This
+document is based on draft-34 of the HTTP/3 I-D [QUIC-HTTP] and draft-21 of the
+QPACK I-D [QUIC-QPACK]. QUIC events are defined in a separate document
+[QLOG-QUIC].
+
+Feedback and discussion welcome at [TODO](TODO). Readers are advised to refer to
```

Comment:
You could still point people to this repo in order to collect anyion call feedback. Then, any and all issues can be brought across to the QUIC WG org when the repo is transferred. And we can update the URL to that new one when the adopted `-00` gets published.